### ISSUE TITLE
Show spinner when creating new file or folder from the menu

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#13161]
### Steps to reproduce
1. Mount SFTP external storage (which is usually a bit slow)
2. Create a new folder or file from the controls bar menu

### Expected result
File/folder entry with spinner should appear.

### Actual result
Nothing happens until the server answers, which might be several seconds.
This is not good UX as the user might think something is broken or nothing is happening.

### Versions
OC 7.0.4 / master

@owncloud/designers what do you think ?
What is best: show spinner in-line inside the list at the correct position ? Or show a spinner in the controls bar while the file/folder is getting created, then auto-scroll to the target position when done ? (a bit like upload)

**Comment by jancborchardt** [comment]
Best would be to both show a spinner inline inside the list at the correct position, as well as autoscroll to that position in the list.

**Comment by jancborchardt** [comment]
@PVince81 8.1 or 8.2?

**Comment by PVince81** [comment]
The code is a bit cleaner now with the new menu and it should be easier to fix this. Keeping for 8.2, hopefully.

**Comment by jancborchardt** [comment]
Yeah, the spinner should show then instead of the filetype icon (file or folder).

**Comment by cmonteroluque** [comment]
This bug will fall off 8.2 if not resolved by 6 Oct
